test/index.test.js

```diff
@@ -1117,4 +1117,225 @@ describe('index', () => {
             });
         });
     });
+
+    describe('_addWebhook', () => {
+        const oauthToken = 'oauthToken';
+        const scmUri = 'bitbucket.org:arnoldWeber/wyattLib:mybranch';
```

Comment:
Is this discussion about the fact that the test currently doesn't have a "valid" `scmUri`?  Why does this matter?  It's not validated.

Here, use this `scmUri`: 
```
bitbucket.org:arnoldWeber/{de7d7695-1196-46a1-b87d-371b7b2945ab}:wyatt
```